src/main/kotlin/org/wycliffeassociates/otter/jvm/app/Main.kt

```diff
@@ -3,25 +3,41 @@ package org.wycliffeassociates.otter.jvm.app
 import org.wycliffeassociates.otter.common.domain.ImportLanguages
 import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
 import org.wycliffeassociates.otter.jvm.app.ui.menu.MainMenu
+import org.wycliffeassociates.otter.jvm.app.ui.menu.MainMenuStylesheet
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.ProjectCreationWizard
+import org.wycliffeassociates.otter.jvm.app.ui.projectpage.view.ProjectPage
+import org.wycliffeassociates.otter.jvm.app.ui.projectpage.view.ProjectPageStylesheet
+import org.wycliffeassociates.otter.jvm.app.ui.viewtakes.view.ViewTakesStylesheet
+import org.wycliffeassociates.otter.jvm.persistence.repositories.AudioPluginRepository
 import tornadofx.*
 import java.io.File
 
 class MyApp : App(Workspace::class) {
     init {
+        importStylesheet(ProjectPageStylesheet::class)
+        importStylesheet(ViewTakesStylesheet::class)
+        importStylesheet(MainMenuStylesheet::class)
         workspace.header.removeFromParent()
         workspace.add(MainMenu())
     }
     override fun onBeforeShow(view:UIComponent) {
-        workspace.dock<ProjectCreationWizard>()
-
+        workspace.dock<ProjectPage>()
     }
 }
 //launch the org.wycliffeassociates.otter.jvm.app
 fun main(args: Array<String>) {
     ImportLanguages(
             File(ClassLoader.getSystemResource("langnames.json").toURI()),
             Injector.languageRepo
-    ).import().subscribe()
+    )
+            .import()
+            .onErrorComplete()
+            .subscribe()
+    Injector
+            .pluginRepository
+            .getAll()
+            .subscribe { plugins ->
+                DefaultPluginPreference.defaultPluginData = plugins.firstOrNull()
+            }
```

Comment:
The injector gets things, leave it at that. Don't chain operations off it just because you can; wrap it in a function that explains what you're doing:

```
configureDefaultPlugins(pluginRepo: PluginRepository) {
  val plugins = pluginRepo.getAll()
  plugins.subscribe { plugins ->
    DefaultPluginPreference.defaultPluginData = plugins.firstOrNull()
  }
}
```